### ISSUE TITLE
include third-party-licenses.json in source distribution

### ISSUE THREAD

**Issue by bollwyvl** [executablebooks/jupyterlab-myst#30]
### Describe the problem/need and solution

**Context**
- as a downstream packager on e.g. https://github.com/conda-forge/staged-recipes/pull/17642, i'd like to include the licenses of vendored JS
- as a user of jupyterlab, i'd like to be able to see what libraries are included in this extension in _Help -> Licenses_

**Problem / Idea**

Conda-forge _strongly encourages_ including licenses for all vendored code: in this case, there's likely a number of libraries that are included in the built JS.

**Solution**

Upgrade to a more recent `@jupyterlab/builder` which generated `third-party-licenses.json`

**Benefit**

- Downstreams would be able to refernce this file
- users would be able to discover the software packages (and licenses) of the underlying JS

### Guide for implementation

From the CLI...

```bash
jlpm upgrade --dev @jupyterlab/builder
```

... or in package.json

```yaml
"devDependencies": {
  "@jupyterlab/builder": "^3.2.0"
}
```

and then

```bash
jlpm
```

### Tasks and updates

- [ ] update `yarn.lock` with newer `@jupyterlab/builder`

**Comment by bollwyvl** [comment]
also, @agoose77 chime in over on https://github.com/conda-forge/staged-recipes/pull/17642 if inclined to co-manage (about the same amount of busy-work as `jupyterlab-markup` :rofl: )

**Comment by agoose77** [comment]
I'm in favour of this idea. I'll get around to it whenever I can, although you're welcome to make a PR (I suspect your time is also rather short on the ground too, though!)

**Comment by agoose77** [comment]
I just re-read this issue, and it's a very simple fix. Fixed in #43
